internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -262,7 +260,7 @@ func (opts *Opts) askClusterOptions() error {
 	clusterName := opts.ClusterName
 	if clusterName == "" {
 		message := ""
-		clusterName = opts.newClusterName()
+		clusterName = autogeneratedName()
```

Comment:
not for this PR but it would be interesting to see if we can set the default of the flag at runtime, like
```go
cmd.Flags().StringVar(&opts.ClusterName, flag.ClusterName, autogeneratedName(), usage.ClusterName)
```

I think I tried something like that before and cobra would not like it but I don't remember what the problem was back then

Reply:
I think this may not work with our logic: now we are asking the cluster name/db user questions when the flags are empty, I am not sure how to adapt this logic with this change (maybe if the `clusterName` starts with `Quickstart-` ask the question?).